contrib/NeonVM/stack.c

```diff
@@ -1,3 +1,7 @@
+#ifdef _MSC_VER
+#pragma warning (push)
+#pragma warning (disable: 4001)     /* Disable single-line comment warning. */
```

Comment:
Why do you need this? I don't see any single line comments in this file.

Reply:
Old code.  it will be removed in a future push.